pyfar/dsp/dsp.py

```diff
@@ -2261,6 +2261,10 @@ def normalize(signal, reference_method='max', domain='time',
         raise ValueError((
             f"domain is '{domain}' and signal is type '{signal.__class__}'"
             " but must be of type 'Signal' or 'FrequencyData'."))
+    if isinstance(signal, (pyfar.TimeData, pyfar.Signal)):
+        if signal.complex and reference_method in ['energy', 'power', 'rms']:
+            raise ValueError(f'{reference_method} is not implemented for '
```

Comment:
Lets maybe give a full list of non-valid methods. Otherwise one might change the norm to get the same error again:
`'The energy, power, and rms reference methods are not implemented for complex signals'`